datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -165,4 +166,10 @@ SdxClusterResponse create(@ValidStackNameFormat @ValidStackNameLength @PathParam
     @ApiOperation(value = "list advertised datalake versions", produces = MediaType.APPLICATION_JSON, nickname = "advertisedruntimes")
     List<AdvertisedRuntime> advertisedRuntimes();
 
+    @POST
+    @Path("/crn/{crn}/ranger_cloud_identity_mapping")
```

Comment:
it looks like you want to do this by env crn instead of cluster crn. You should change the path and the parameter name. see getByEnvCrn method above for comparison.